index.html

```diff
@@ -147,8 +147,31 @@ <h2 id="skill">Skills</h2>
             </div>
 
         </section>
+
+        <section id="Contact-us">
+            <h4 class="interest">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+            <div class="form1">
+            <form action="https://formspree.io/f/mnqwkpaw" method="POST">
+              <input class="text1" type="text" name="name88" id="name89" placeholder="Full name" required>
+              <br>
+              <input class="text1" type="email" name="email" id="email" placeholder="Email address" required>
+              <br>
+              <input class="text2" type="text" name="text" id="text" placeholder="Enter text here" maxlength="500">
```

Comment:
- [ ] Kindly add a `maxlength="30"`  to the name input
- [ ] Please, message input should be `<textarea>`, it should be required too.